models/agol.js

```diff
@@ -352,7 +352,7 @@ var AGOL = function (koop) {
       if (err) return callback(err)
       info = info || {}
       info.status = 'Failed'
-      info.retrieved_at = new Date()
+      info.retrieved_at = Date.now()
```

Comment:
Just want to raise a question about how timestamps are dealt with in Koop because `new Date()` returns a Date object whereas `Date.now()` returns a Number. We can settle this in another GH issue and/or PR